monkey/monkey_island/cc/models/monkey.py

```diff
@@ -8,7 +8,7 @@
 from monkey_island.cc.models.monkey_ttl import MonkeyTtl, create_monkey_ttl_document
 from monkey_island.cc.consts import DEFAULT_MONKEY_TTL_EXPIRY_DURATION_IN_SECONDS
 from monkey_island.cc.models.command_control_channel import CommandControlChannel
-from monkey_island.cc.utils import local_ip_addresses
+from monkey_island.cc.network_utils import local_ip_addresses
```

Comment:
Ugh this should be in common. Either move yourself or I'll see about rolling this into its own PR

Reply:
I think we can keep it here until monkey needs to use it also